src/test/QldbDriver.test.ts

```diff
@@ -211,7 +210,7 @@ describe("QldbDriver", () => {
             const result = await chai.expect(qldbDriver.executeLambda(lambda, defaultRetryConfig)).to.be.rejected;
             chai.assert.equal(result.code, errorCode);
             sinon.assert.callCount(executeLambdaSpy, defaultRetryConfig.getRetryLimit() + 1);
-        });
+        }).timeout(20000);
```

Comment:
Chai has a default timeout for tests of 5000ms, and since this is a quadruple retry it can exceed it.

Reply:
This is a long time for a test to run. Rather change the timeout time then this can be a small number again

Reply:
I reduced the retry attempts allowed on the test and removed the timeout modifier